ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -120,8 +114,9 @@ void requestBannerAd(@NonNull AdSize adSize, @NonNull MediationAdRequest mediati
       config = AdapterParametersParser.parse(mediationExtras, serverParameters);
     } catch (IllegalArgumentException exception) {
       Log.w(TAG, "Failed to load ad from Vungle.", exception);
-      if (bannerListener != null) {
-        bannerListener.onAdFailedToLoad(adapter, AdRequest.ERROR_CODE_INVALID_REQUEST);
+      MediationBannerListener listener = bannerListener.get();
+      if (listener != null && adapter.get() != null) {
```

Comment:
same here... but it can't become null here as it's called right after the constructor

Reply:
done.